modules/profiles/modules.php

```diff
@@ -133,7 +133,16 @@ public function process() {
             $profile['id'] = $form['profile_id'];
             Hm_Profiles::edit($form['profile_id'], $profile);
         } else {
+            $profiles = $this->get('profiles');
+
+        foreach ($profiles as $existing_profile) {
+            if ($existing_profile["smtp_id"] === $profile["smtp_id"]) {
```

Comment:
Here you are testing with smtp_id, this will prevent from having many profiles even with different email on that smtp. If the idea is to prevent duplication, use the email field 'replyto' and 'address' instead.